<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard_1</class>
 <widget class="QDialog" name="dashboard_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Routine Management System</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>841</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>841</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_sun">
      <property name="text">
       <string>Sunday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_mon">
      <property name="text">
       <string>Monday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_tue">
      <property name="text">
       <string>Tuesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_wed">
      <property name="text">
       <string>Wednesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_thu">
      <property name="text">
       <string>Thursday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_friday">
      <property name="text">
       <string>Friday</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox_chooseCourse">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Choose Routine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Civil I</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_updateTable">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>20</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Table</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
